pkgs/development/interpreters/python/cpython/default.nix

```diff
@@ -104,6 +104,8 @@ in with passthru; stdenv.mkDerivation {
     (
       if isPy35 then
         ./3.5/python-3.x-distutils-C++.patch
+      else if isPy37 then
+        ./3.7/python-3.x-distutils-C++.patch
```

Comment:
We can't fetch this from somewhere?

Reply:
@LnL7 We used to upload those to the python bug tracker. But it's done inconsistently with unclear naming and the changes to the patches that we are providing are not very meaningful, so we might be better of maintaining those in our tree.

Reply:
Ok. Tested a few packages, this can go straight to master since it's broken at the moment.